docs/components/select.md

```diff
@@ -401,8 +403,10 @@ You can pass any input attribute and it will get properly bound to the element.
 
 You can use the `item-template` slot to customize the look and feel of your items. Use slots to gain access to the `item` data.
 
+If you use the `select-item-label` and `select-item-desc` classes within the slot as shown in the example below, the dropdown items will inherit preconfigured styles for two-level select items which you're then free to customize.
+
 <div>
-  <KSelect :items="myItems" width="100%" :filterFunc="customFilter">
+  <KSelect appearance='select' :items="myItems" width="100%" :filterFunc="customFilter">
```

Comment:
We actually don't want to convert this to `select` appearance here because filtering is not turned on by default for it (and this is example code for `customFiltering`).

Reply:
Yeah, I was trying to make it look similar to the example for geo select, @adamdehaven : what do you think ?

Reply:
That sounds good to me! @kaiarrowood : what's your thought on this ?

Reply:
Ok.. removing `select`

Reply:
I don't particularly care either way lol, just if there is `filterFunc` in the example either have it be `dropdown` appearance or `select` appearance + `enable-filtering` prop